usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -50,6 +50,16 @@ def add_arguments(self, parser):
             default=os.path.dirname(os.path.abspath(__file__)),
             type=str,
             help='Set for a custom location of output files')
+        parser.add_argument(
+            '--index_name',
+            type=str,
+            help='Set an index name to ingest data into',
+            default='staging_transactions')
+        parser.add_argument(
+            '--previous_index',
```

Comment:
Are you expecting the user running this script to know what the previous index name is? How do you propose we automate this?

Reply:
We could add date prefixes and pick the min date for the index pattern and assume that is the one we want to delete. Yes this assumes.